src/PactNet/Interop/StringResult.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;

```

Comment:
Some formatting issues in this file:

- Needs a blank line after the imports
- Names contain underscores, which doesn't match C# naming conventions
- Fields named `0` aren't descriptive